src/traces/bar/attributes.js

```diff
@@ -42,7 +42,16 @@ var marker = extendFlat({
         editType: 'style',
         description: 'Sets the opacity of the bars.'
     },
-    pattern: pattern
+    pattern: pattern,
+    cornerradius: {
+        valType: 'any',
+        dflt: 0,
```

Comment:
This `dflt` should be unused if `trace.cornerradius` depends of `layout.barcornerradius`.

Reply:
@archmoj would the correct approach be to just remove the `dflt`?

Reply:
Exactly. Please remove and mention the `barcornerradius` default in the description.
Thank you!